src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/dao/LimitedPartnershipIncorporationDao.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.model.dao;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+@Document(collection = "limited_partnership_submissions")
```

Comment:
in light of what Mike said earlier about having different collections, just wondering if the collection name should be limited_partnership_incorporations or similar

Reply:
I thought about this. Wondering if that should be done as a separate piece as ideally the current collection name will need to change as well. Or if we're agreed on limited_partnership_incorporations I can make the change now.

Reply:
Updated to use new collection:  limited_partnership_incorporations